instruments/instruments.go

```diff
@@ -6,6 +6,10 @@ const Path = "instruments"
 
 type InstrumentType string
 
+const Validation = "/validation/bank-accounts/"
```

Comment:
This could be named as `ValidationPath`. And move up, just after `const Path = "instruments"`

Reply:
👍